KlingonAssistant/data/mem-16-t.xml

```diff
@@ -1570,8 +1570,8 @@ This dialect is sort of half-way between the standard dialect and the {Qotmagh S
       <column name="_id">32205</column>
       <column name="entry_name">tellar wa'</column>
       <column name="part_of_speech">n:place,fic</column>
-      <column name="definition">Tellar Prime</column>
-      <column name="definition_de"></column>
+      <column name="definition">Tellar Prime (planet name)</column>
```

Comment:
Actually, "planet name" is usually not in the definition unless there's ambiguity between the planet and a person (like "Vulcan"). I added this to "Donatu V" because I was just typing what's in the booklet, but now that I look at it, that's not part of the definition. I'll remove it from here, and there too.